daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ledgerinteraction/ScriptLedgerClient.scala

```diff
@@ -208,6 +208,17 @@ trait ScriptLedgerClient {
       mat: Materializer,
   ): Future[Option[List[UserRight]]]
 
+  def trySubmit(
+      actAs: OneAnd[Set, Ref.Party],
+      readAs: Set[Ref.Party],
+      commands: List[command.ApiCommand],
+      optLocation: Option[Location],
+  )(implicit
+      ec: ExecutionContext,
+      mat: Materializer,
+  ): Future[Either[SubmitError, Seq[ScriptLedgerClient.CommandResult]]] =
+    throw new java.lang.UnsupportedOperationException("I'm not ready")
```

Comment:
~~Hello not ready, I'm dad~~

Even considering this is transient, maybe "`trySubmit` is not yet implemented in the `ScriptLedgerClient`" would be more helpful.